sw/airborne/subsystems/intermcu/intermcu_fbw.c

```diff
@@ -150,6 +151,13 @@ static void intermcu_parse_msg(void (*commands_frame_handler)(void))
       commands_frame_handler();
       break;
     }
+    case DL_IMCU_TELEMETRY: {
```

Comment:
This needs to be protected, so it still compiles/works if you don't use telemetry_intermcu